sys/registry/namespace/sys/definitions/0001_rgb_led.yaml

```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=../../../../../dist/tools/registry_gen/schema.json
+id: 1
+name: rgb
+description: Representation of an rgb color.
+items:
+  - id: 0
+    name: red
+    description: Intensity of the red color of the rgb lamp.
+    type: uint8
+
+  - id: 1
+    name: green
+    description: Intensity of the green color of the rgb lamp.
+    type: uint8
+
+  - id: 2
+    name: blue
+    description: Intensity of the blue color of the rgb lamp.
+    type: uint8
+
+  - id: 3
+    name: brightnesses
```

Comment:
I'd go with the singular form as you also did in the description

Reply:
I removed the brightnesses attogether because it is not really part of an RGB LED anyways.